.github/workflows/release_vsyncer.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and release vsyncer
+
+on:
+  release:
+    types:
+      - published
+      - edited
+
+jobs:
+  build-release:
+    runs-on: ubuntu-latest
```

Comment:
Do go apps link against glibc? In that case you might want to use the **oldest** ubuntu version available, to support as many ubuntu versions as possible

Reply:
No, in fact, we could generate a fully static tool with -static option. That would make the vsyncer very independent of the Linux OS version.